File: 1-Iniciante/1160.cpp
#include <iostream>

using namespace std;

int main(void) {
    int i, j, anos = 0, qnt;
    int PA, PB;
    double G1, G2;

    cin >> qnt;

    for (j = 0; j < qnt; ++j) {
        cin >> PA >> PB >> G1 >> G2;

        i = 1;
        while (1) {
            PA += (double)((G1 / 100.0) * PA);
            PB += (double)((G2 / 100.0) * PB);
            if (PA > PB || i == 101) break;
            ++i;
        }

        if (i == 101) cout << "Mais de 1 seculo." << endl;
        else cout << i << " anos." << endl;
    }

    return 0;
}

File: 1-iniciante/1160.py
n = int(input())
for g in range(n):
    e = input().split()
    a = int(e[0])
    b = int(e[1])
    ca = float(e[2])
    cb = float(e[3])
    q = 0
    while a <= b:
        a += int(a * ca / 100)
        b += int(b * cb / 100)
        q += 1
        if q == 101: break
    if q <= 100: print(q, 'anos.')
    else: print('Mais de 1 seculo.')
